packages/walletconnect/src/index.ts

```diff
@@ -49,17 +62,17 @@ export class WalletConnect extends Connector {
 
   private readonly timeout: number
 
-  private eagerConnection?: Promise<MockWalletConnectProvider>
+  private eagerConnection?: Promise<WalletConnectProvider>
 
   constructor({ actions, options, defaultChainId, timeout = 5000, onError }: WalletConnectConstructorArgs) {
```

Comment:
Can you extract the default timeout to a `const DEFAULT_TIMEOUT = 5000` at the top of the file?